perllib/FixMyStreet/SendReport/Open311.pm

```diff
@@ -144,14 +144,11 @@ sub send {
             $self->success( 1 );
         } else {
             $result *= 1;
-            # temporary fix to resolve some issues with west berks
-            if ( $row->bodies_str =~ /2619/ ) {
-                $result *= 0;
-            }
+            $self->error( "Failed to send over Open311\n" ) unless $self->error;
+            $self->error( $self->error . "\n" . $open311->error );
```

Comment:
It should be the empty string, from `has error => ( is => 'rw', 'isa' => Str, default => '' );` so it should only add a newline at present.